client/src/test/java/com/arcbees/beestore/client/application/product/ProductPresenterTest.java

```diff
@@ -34,6 +38,8 @@
 import static org.mockito.Mockito.reset;
 import static org.mockito.Mockito.verify;
 
+import static com.gwtplatform.dispatch.rest.delegates.test.DelegateTestUtils.givenDelegate;
```

Comment:
if I take a look at the ArcBees checkstyle, static imports follow the same order as regular imports.